packages/grafana-schema/src/common/mudball.cue

```diff
@@ -178,7 +178,7 @@ OptionsWithTextFormatting: {
 } @cuetsy(kind="interface")
 
 // TODO docs
-BigValueColorMode: "value" | "background" | "none" @cuetsy(kind="enum")
+BigValueColorMode: "value" | "backgroundSolid" | "backgroundGradient" | "none" @cuetsy(kind="enum")
```

Comment:
by changing "background" to "backgroundGradient" you kick off a whole migration process (that is not supported in this PR)... if we just want to add a solid option, you can keep background and add 'backgroundSolid"

Reply:
Yeah, I saw that things were breaking. I was looking into where we've done a migration like this before in the code; but do you think it's worth it, or just keep `background` like you mentioned before?